staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -244,6 +248,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		openAPIV3Config:            c.GenericConfig.OpenAPIV3Config,
 		proxyCurrentCertKeyContent: func() (bytes []byte, bytes2 []byte) { return nil, nil },
 		rejectForwardingRedirects:  c.ExtraConfig.RejectForwardingRedirects,
+		tracerProvider:             c.GenericConfig.TracerProvider,
```

Comment:
isn't that preserved in GenericAPIServer above?

Reply:
If yes, there is no need to plumb it through manually.

Reply:
i could definitely be missing it, but i couldn't find a way to access the TracerProvider

Reply:
it only appears to be used here: https://github.com/kubernetes/kubernetes/blob/9791f0d1f39f3f1e0796add7833c1059325d5098/staging/src/k8s.io/apiserver/pkg/server/config.go#L983

Reply:
Yeah this is only accessible from GenericConfig. I think it's fine to plumb it through?